package.json

```diff
@@ -134,6 +134,7 @@
   "devDependencies": {
     "gulp-standard": "^8.0.3",
     "gulp-tape": "0.0.9",
+    "sinon": "^1.17.7",
```

Comment:
is this (and all the other dev dependencies) going to cause `HMRC prototype kit` problems? May be worth moving them to `dependencies` but as a separate PR?